source/Templates/NewStep/Sashimi.NamingIsHard/source/Sashimi/MyActionHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Sashimi.Server.Contracts.ActionHandlers;
+using Sashimi.Server.Contracts.Calamari;
+
+namespace Sashimi.NamingIsHard
+{
+    class MyActionHandler : IActionHandler
+    {
+        public IActionHandlerResult Execute(IActionHandlerContext context)
+        {
+            return context.CalamariCommand(new CalamariFlavour("Calamari.NamingIsHard"), "my-command-name")
+                .Execute();
+        }
+
+        public string Id { get; } = "NamingIsHard";
```

Comment:
The action ID/Names shouldn't be replaced by the template engine

Reply:
Fixed